CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+## Releases
+
+Releases are created by pushing a tag on `main` or `release-*` branch:
+
+```
+git tag -a 0.0.3 -m "Release 0.0.3"
+git push --tags
+```
+
+This kicks off a GitHub Action which automatically publishes the package to PyPi.
+
+When that completes, grab the hash from [pypi](https://pypi.org/project/saturn-client/#files)
+and open a PR to bump that and the version in the conda recipe.
+
+As of writing this, building and uploading the conda package is still a manual process
+that @hhuuggoo does.
```

Comment:
Is this still true? For this or dask-saturn?

Reply:
well it's often me, but yes.

Reply:
Maybe we shouldn't bother with conda packages....

Reply:
+100

Reply:
huh, I thought there was support in release-images for that.

Reply:
There is, but it's only used by `jupterlab_saturn` right now. We can add more to it, but that would tie it to our release cycles.

If we can get away with only having packages in PyPI and not worry about conda repos, I'm 1000% for it.

Reply:
Yeah these are pure python libraries so we can just do pypi and call it a day

Reply:
Got rid of it